src/components/calculator.css

```diff
@@ -0,0 +1,45 @@
+.calculator-container {
+  background-color: #1a1a1a;
+  color: white;
+  width: 30rem;
+  height: 30rem;
+  display: grid;
+  grid-template-rows: 1fr 4fr 1fr;
+}
```

Comment:
- [ ] [OPTIONAL] Your calculator looks nice as in the given design 👍. I'd suggest you put it to the center of the page to make it better 💡

Reply:
Thank you for your review! I have centered the calculator using flexbox.